<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputNum</class>
 <widget class="QDialog" name="InputNum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>350</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>350</height>
    </size>
   </property>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>275</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>275</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>179</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>125</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>0</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_4">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>0</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>275</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>275</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>tab</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>125</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>125</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>0</y>
      <width>25</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>25</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>25</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>5752752</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
